public/content-script.js

```diff
@@ -0,0 +1,140 @@
+// разработка попап
+
+const takeIt = document.createElement('div');
+
+//разработка логотипа с стилями
+
+const logo = document.createElement('h1');
+logo.innerText = "Take Word";
+logo.style.fontSize = '46px';
+logo.style.fontFamily = 'monospace';
+logo.style.fontVariant = 'petite-caps';
+logo.style.fontStyle = 'oblique';
+logo.style.fontWeight = 'revert';
+logo.style.color = 'gold';
+
+// стили попап
+
+takeIt.style.position = 'absolute';
+takeIt.style.top = '10px';
+takeIt.style.right = '10px';
+takeIt.style.height = '400px';
+takeIt.style.width = '250px';
+takeIt.style.zIndex = '9999';
+takeIt.style.backgroundColor = 'black';
+takeIt.style.display = 'flex';
+takeIt.style.flexDirection = 'column';
+takeIt.style.justifyContent = 'space-around';
+takeIt.style.alignItems = 'center';
+takeIt.style.border = 'solid';
+takeIt.style.overflow = "hidden";
+takeIt.style.boxSizing = 'border-box'
+
+// разработка дива с английским словом внутри
+
+const wordDiv = document.createElement('div');
+const word = document.createElement('h1');
+word.innerText = "";
+wordDiv.appendChild(word);
+
+// стили дива с английским словом
+
+wordDiv.style.border = 'solid';
+wordDiv.style.width = '100%';
+wordDiv.style.height = '40%';
+wordDiv.style.backgroundColor = 'gold';
+wordDiv.style.display = 'flex';
+wordDiv.style.justifyContent = 'center';
+wordDiv.style.alignItems = 'center';
+
+// стили английского слова
+
+word.style.fontSize = '35px'
+word.style.fontVariant = 'all-petite-caps'
+word.style.fontWeight = 'revert';
+
+// разработка дива с переводом внутри
+
+const translDiv = document.createElement('div');
+const transl = document.createElement('h1');
+transl.innerText = "";
+translDiv.appendChild(transl);
+
+// стили дива с переводом
+
+translDiv.style.border = 'solid';
+translDiv.style.width = '100%';
+translDiv.style.height = '40%';
+translDiv.style.backgroundColor = 'gold';
+translDiv.style.display = 'flex';
+translDiv.style.justifyContent = 'center';
+translDiv.style.alignItems = 'center';
+
+//стили текста перевода
+
+transl.style.fontSize = '28px'
+transl.style.fontVariant = 'all-petite-caps'
+transl.style.fontWeight = 'revert';
+translDiv.appendChild(transl);
+
+// кнопка тейк-ит
+
+const button = document.createElement('button');
+button.innerText = 'Take it!';
+
+// стили кнопки
+
+button.style.color = 'gold';
+button.style.width='100%';
+button.style.backgroundColor='black';
+button.style.border='solid 2px';
+button.style.fontSize='large';
+button.style.fontFamily = 'monospace';
+button.style.fontVariant = 'petite-caps';
+button.style.fontWeight = 'bolder ';
+
+// пихаем все в попап, а сам попап в ДОМ
+
+takeIt.appendChild(logo);
+takeIt.appendChild(wordDiv);
+takeIt.appendChild(translDiv);
+takeIt.appendChild(button);
+
+document.getElementsByTagName('body')[0].appendChild(takeIt)
+
+// функция получения рандомного слова
+
+takeIt.getRandomWord = async () => {
```

Comment:
move this to api.js you can import this by stacking script tags in html on top of each other